ImplementationGuide/markdown/Datenobjekte/Profile_AMTSBewertung.md

```diff
@@ -90,42 +80,56 @@ Folgende Suchparameter sind für das Bestätigungsverfahren relevant, auch in Ko
+1. Der Suchparameter "patient" MUSS unterstützt werden:
 
    Beispiele:
 
-   ```GET [base]/RiskAssessment?encounter.identifier=http://mein-krankenhaus.example/fhir/sid/fallnummern|7567867```
+    ```GET [base]/RiskAssessment?patient=Patient/123```
 
-	 ```GET [base]/RiskAssessment?encounter.identifier=7567867```
+    Anwendungshinweise: Weitere Informationen zur Suche nach Reference-type Parametern finden sich in der [FHIR-Basisspezifikation - Abschnitt "Reference Search"](https://www.hl7.org/fhir/R4/search.html#reference).
 
-   Anwendungshinweise: Weitere Informationen zur Suche nach Reference-type Parametern finden sich in der [FHIR-Basisspezifikation - Abschnitt "Reference Search"](https://www.hl7.org/fhir/R4/search.html#reference).
+1. Der Suchparameter "risk" MUSS unterstützt werden:
 
-	 Weitere Informationen zur Suche nach verketteten Parametern finden sich in der [FHIR-Basisspezifikation - Abschnitt "Chained Parameters"](https://hl7.org/fhir/R4/search.html#chaining).
+    Beispiele:
 
-1. Der Suchparameter "patient" MUSS unterstützt werden:
+    ```GET [base]/RiskAssessment.risk=http://terminology.hl7.org/CodeSystem/risk-probability|high```
 
-   Beispiele:
+    Anwendungshinweise: Weitere Informationen zur Suche nach Token-type Parametern finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](https://hl7.org/fhir/R4/search.html#token).
 
-    ```GET [base]/RiskAssessment?patient=Patient/123```
+---
 
-    Anwendungshinweise: Weitere Informationen zur Suche nach Reference-type Parametern finden sich in der [FHIR-Basisspezifikation - Abschnitt "Reference Search"](https://www.hl7.org/fhir/R4/search.html#reference).
+### Verkettete Suchparameter (Chaining und Reverse Chaining)
 
-1. Der verkettete Suchparameter "patient.identifier" MUSS unterstützt werden:
+Auch die verketteten Suchparameter MÜSSEN unterstützt werden und sind mit entsprechenden Referenzen im CapabilityStatement unter ```CapabilityStatement.rest.resource.searchInclude``` bzw. ```CapabilityStatement.rest.resource.searchRevInclude``` angegeben. Siehe {{pagelink:ImplementationGuide/markdown/CapabilityStatement.md}}
 
-    Beispiele:
+Informationen und Beispiele zur Suche nach verketteten Parametern finden sich [im Basismodul](https://simplifier.net/guide/isik-basis-v4/UebergreifendeFestlegungen-UebergreifendeFestlegungen_Suchparameter) und in der FHIR-Basisspezifikation im [Abschnitt "Chained Parameters"](https://hl7.org/fhir/R4/search.html#chaining) und im [Abschnitt "Reverse Chaining"](https://hl7.org/fhir/R4/search.html#has).
+
+Die verketteten Suchparameter des Profils ```RiskAssessment``` hängen wie folgt mit den festgelegten {{pagelink:ImplementationGuide/markdown/UseCasesAnwendung/UseCases.md, text:Anwendungsfällen (Use Cases)}}  zusammen:
 
-    ```GET [base]/RiskAssessment?patient.identifier=http://mein-krankenhaus.example/fhir/sid/patienten|1032702```
+1. Der verkettete Suchparameter ```Patient:identifier``` unterstützt den Anwendungsfall:
```

Comment:
patient:identifier, patient.identifier oder beides?

In jedem Fall stimmen Text und Beispiel nicht überein

Reply:
angepasst